### DISCUSSION TITLE
Use egg ns code to upgrade skyline emulator

### DISCUSSION THREAD

**Discussion by Beastboysubb** [skyline-emu/skyline#1561] [Ideas]
It helps a lot

**Comment by Swizzlefowl** [comment]
Do you mean use Yuzu's code?

**Reply by Beastboysubb** [reply]
Yes

**Reply by Allatel** [reply]
Wait... Skyline already has yuzu's devs help...

**Reply by yhak22** [reply]
The performance in skyline is more stable than shity egg ns

**Comment by hope3x** [comment]
Is this a joke?

**Comment by Bossplay228** [comment]
this is a bad idea because on my phone egg ns can’t run anything except Sonic and that’s it, but skyline launches cuphead, hollow knight, and so on for me

**Comment by PixelyIon** [comment]
Well, there's not much to say here except that this is an unbelievably stupid suggestion.
